test/GraphingSpec.hs

```diff
@@ -12,7 +12,7 @@ spec = do
   describe "unfold" $ do
     it "should unfold deeply" $ do
       let graph :: Graphing Int
-          graph = unfold [10] (\x -> if x > 0 then [x -2] else []) id
+          graph = unfold [10] (\x -> if x > 0 then [x - 2] else []) id
```

Comment:
CI's code formatter version is not pinned - new version (0.4) found these formatting errors. Although out of scope for the issue - fixing them in here

Reply:
CI's code formatter is definitely pinned, but it WAS just updated within the last few days.  You can watch [fossas/haskell-dev-tools](https://github.com/fossas/haskell-dev-tools) to get updates.